lib/src/model/movies_model.dart

```diff
@@ -0,0 +1,37 @@
+import 'movies_result_model.dart';
+
+class MoviesModel {
+  late int _page;
+  late int _totalResults;
+  late int _totalPages;
+  late List<MoviesResultModel> _results = [];
+
+  MoviesModel.fromJson(Map<String, dynamic> parsedJson) {
+    _page = parsedJson['page'];
+    _totalResults = parsedJson['total_results'];
+    _totalPages = parsedJson['total_pages'];
+
+    List<MoviesResultModel> temp = [];
+    for (int i = 0; i < parsedJson['results'].length; i++) {
+      temp.add(
+        MoviesResultModel(
+          parsedJson['results'][i],
+        ),
+      );
+    }
+    _results = temp;
+  }
+
+  List<MoviesResultModel> get results => _results;
+
+  int get totalPages => _totalPages;
+
+  int get totalResults => _totalResults;
+
+  int get page => _page;
+
+  @override
+  String toString() {
+    return 'MoviesModel{_page: $_page, _totalPages: $_totalPages, _totalResults: $_totalResults, _results: $_results}';
+  }
```

Comment:
Remove this function, if you dont use it

Reply:
done